### DISCUSSION TITLE
Github Achievements

### DISCUSSION THREAD

**Discussion by ihsandedec** [community/community#20760] [New to GitHub] [answered]
How to earn Achievements on github.

**Comment by Jaymoney7738** [comment]
Sur"did

**Comment by thuongtruong1009** [comment, marked as answer]
You can easily get achievements to badge on GitHub, reference at:
[Collection for beginning at here](https://github.com/Schweinepriester/github-profile-achievements)
[My discussions at community](https://github.com/github-community/community/discussions/20443)
[My GitHub profile](https://github.com/thuongtruong1009)

**Reply by Jaymoney7738** [reply]
Thank you

**Comment by ihsandedec** [comment]
thank you :)

**Comment by thuongtruong1009** [comment]
Hi, @ihsandedec !
Can you mark my comment as an answer to help me get the badge and help more people can find the information keyword!
Thank you very much!
________________________________
Từ: İhsan Dedeç ***@***.***>
Đã gửi: 17 Tháng Bảy 2022 10:09 CH
Đến: github-community/community ***@***.***>
Cc: TRAN NGUYEN THUONG TRUONG ***@***.***>; Comment ***@***.***>
Chủ đề: Re: [github-community/community] Github Achievements (Discussion #20760)


thank you :)

**Reply by ihsandedec** [reply]
this is my first time using this platform, so I couldn't be sure. all right

**Reply by thuongtruong1009** [reply]
> this is my first time using this platform, so I couldn't be sure. all right
Thank you very much! You're handsome ❤
Thank you very much! Have a nice day!

**Reply by ihsandedec** [reply]
I'm sending a tracking request. I would be very glad if you would agree. :)

**Reply by thuongtruong1009** [reply]
Yes, of course! Thank you very much!
